### DISCUSSION TITLE
RangeError: Invalid content for node listItem: <>, on inserting an empty bullet point "-" for markdown

### DISCUSSION THREAD

**Discussion by rarchitgupta** [steven-tey/novel#396] [Q&A]
## Streaming an LLM response into the Novel editor causes this issue

I have configured the tiptap-markdown extension to accept markdown input into the editor like so:
```typescript
import { Markdown } from 'tiptap-markdown';

const editor = new Editor({
    content: "# Title",
    extensions: [
        StarterKit,
        Markdown,
    ],
});
```

I receive content from a stream like so in small bits (Langchain stream) and I accumulate the response in a variable. At the end of the loop, I insert the content into the editor. This works completely fine:
```typescript
let accumulatedResponse = "";
for await (const chunk of stream) {
   if (typeof chunk == "string") {
        accumulatedResponse += chunk;
    }
}
editor.commands.insertContent(accumulatedResponse);
```

However when I try to insert the command inside the loop (This is essential to give it that streaming behaviour):
```typescript
for await (const chunk of stream) {
    if (typeof chunk == "string") {
        editor.commands.insertContent(chunk);
    }
}
```
I get the error:
```
RangeError: Invalid content for node listItem: <>
```

This can also be simulated by doing a simple:
```typescript
editor.commands.insertContent("-")
```

I've tried accumulating more chunks per loop instead of inserting them one by one but it hasn't worked for me. And maintaining this streaming behaviour is important

Any help here would really be appreciated

**Comment by rarchitgupta** [comment]
@andrewdoro I believe you added markdown support in another PR, perhaps you could shed some light here

**Reply by andrewdoro** [reply]
hey @rarchitgupta  I am pretty sure tiptap-markdown doesn't support streaming, that's why you get this error
https://github.com/steven-tey/novel/discussions/7#discussioncomment-6301035
